src/components/HeadCellMenu/__tests__/HeadCellMenu.test.tsx

```diff
@@ -0,0 +1,313 @@
+/* eslint-disable no-underscore-dangle */
+import React from 'react';
+import '@testing-library/jest-dom';
+import { stardust } from '@nebula.js/stardust';
+import { render, fireEvent, screen, waitFor, waitForElementToBeRemoved } from '@testing-library/react';
+import HeadCellMenu from '../HeadCellMenu';
+import * as useFieldSelectionHook from '../use-field-selection';
+import { HeaderData, MenuAvailabilityFlags, UseFieldSelectionOutput } from '../types';
+
+jest.mock('../use-field-selection', () => ({
+  __esModule: true,
+  ...jest.requireActual('../use-field-selection'),
+}));
+
+type ExtendedEmbed = stardust.Embed & {
+  __DO_NOT_USE__: {
+    popover: () => void;
+  };
+  on: () => void;
+};
+
+describe('<HeadCellMenu />', () => {
+  let app: EngineAPI.IApp;
+  let headerData: HeaderData;
+  let translator: stardust.Translator;
+  let translatorGetMock: jest.Mock<() => string>;
+  let menuAvailabilityFlags: Partial<Record<MenuAvailabilityFlags, boolean>>;
+  let sortFromMenu: jest.Mock<Promise<void>>;
+  let changeActivelySortedHeader: jest.Mock<Promise<boolean>>;
+  let interactions: stardust.Interactions;
+  let anchorRef: React.RefObject<HTMLDivElement>;
+  let setFocusOnClosetHeaderAdjuster: jest.Mock<() => void>;
+  let listboxRef: React.RefObject<HTMLDivElement>;
+  let handleHeadCellMenuKeyDown: jest.Mock<() => void>;
+
+  let embed: ExtendedEmbed;
+  let defaultListboxAnchorOpts: any;
+  let fieldInstanceMock: EngineAPI.IField;
+  let selectionActionsEnabledStatusMock: Record<string, boolean>;
+  let resetSelectionActionsEnabledStatusMock: jest.Mock<any, any>;
+  let updateSelectionActionsEnabledStatusMock: jest.Mock<any, any>;
+  let model: EngineAPI.IGenericObject;
+  let useFieldSelectionHookResult: UseFieldSelectionOutput;
+  const menuLabels = [
```

Comment:
MAybe these could be hard-coded constants use by both code and test? So that when the `SNTable` prefix is removed both code and tests gets updated

Reply:
Actually these prefixes are changed in translation PR ([this is one example of changed ones](https://github.com/qlik-oss/nebula-table-utils/blob/1132b21f2e6be595dd13946bf1f320240ac0d644/src/components/HeadCellMenu/utils.ts#L61)), but i'm not sure if we can extract them to constants, because in test cases we just check for their text (batch them into an array and iterate and expect), but in utils file we use them separately/onebyone in different lines.